static/app/components/tooltip.tsx

```diff
@@ -320,67 +240,56 @@ function Tooltip({
     return <Fragment>{children}</Fragment>;
   }
 
-  const modifiers: PopperJS.Modifiers = {
-    hide: {enabled: false},
-    preventOverflow: {
-      padding: 10,
-      enabled: true,
-      boundariesElement: 'viewport',
-    },
-    applyStyle: {
-      gpuAcceleration: true,
-    },
-  };
-
-  const visible = forceShow || isOpen;
-
-  const tip = visible ? (
-    <Popper placement={position} modifiers={modifiers}>
-      {({ref, style, placement, arrowProps}) => (
-        <PositionWrapper style={style}>
-          <TooltipContent
-            id={tooltipId}
-            initial={{opacity: 0}}
-            animate={{
-              opacity: 1,
-              scale: 1,
-              transition: testableTransition({
-                type: 'linear',
-                ease: [0.5, 1, 0.89, 1],
-                duration: 0.2,
-              }),
-            }}
-            exit={{
-              opacity: 0,
-              scale: 0.95,
-              transition: testableTransition({type: 'spring', delay: 0.1}),
-            }}
-            style={computeOriginFromArrow(position, arrowProps)}
-            transition={{duration: 0.2}}
-            className="tooltip-content"
-            aria-hidden={!visible}
-            ref={ref}
-            data-placement={placement}
-            popperStyle={popperStyle}
-            onMouseEnter={() => isHoverable && handleOpen()}
-            onMouseLeave={() => isHoverable && handleClose()}
-          >
-            {title}
-            <TooltipArrow
-              ref={arrowProps.ref}
-              data-placement={placement}
-              style={arrowProps.style}
-            />
-          </TooltipContent>
-        </PositionWrapper>
-      )}
-    </Popper>
-  ) : null;
-
   return (
     <Manager>
       <Reference>{({ref}) => renderTrigger(children, ref)}</Reference>
-      {createPortal(<AnimatePresence>{tip}</AnimatePresence>, tooltipPortal)}
+      {createPortal(
+        <AnimatePresence>
+          {forceShow || isOpen ? (
```

Comment:
Also 👎 to removing the interim `visible` variable, the name provides some contextual value as to what it's for. `forceShow` and `isOpen` isn't far off for sure, but I think `visible` makes things nice and clear.

Reply:
I'm 50/50 on this tbh. Deferring to other opinions if anyone has them, otherwise I'll revert

Reply:
🤔 
I don't hate having an `isVisible` variable for the context either 🤷 

I prefer to name it `isVisible` as opposed to `visible` though. slightly more descriptive of its purpose. 
similarly any other boolean things would be named `has_` `is_` etc.

Also no strong opinions on whether to keep a variable or not tho

Reply:
👍  keep `visible`

Reply:
@k-fish @nhsiehgit I think let's take the prop to maybe change the prop from "open" to visible then, so we dont use two different names :)